spectrum-discover/spectrum-discover.ibm.com_v1alpha1_spectrumdiscover_cr.yaml

```diff
@@ -2,188 +2,9 @@ apiVersion: spectrum-discover.ibm.com/v1alpha1
-
-  image:
-    repository: $image_repository/
-
-  node_select_key:
-
-  maintenance: 'off'
-  apiserver:
-    replicas: 1
-
-  keystone:
-    replicas: 1
-
-  auth:
-    replicas: 1
-
-  connmgr:
-    replicas: 1
-
-  consumer:
-    ceph-le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    cos-le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    cos-scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    file-scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    protect-scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    scale-le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-    scale-scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      group_id: mo1
-      replicas: 10
-
-  contentsearchagent:
-    replicas: 1
-
-  dbwhrest:
-    replicas: 1
-
-  importtags:
-    replicas: 1
-
-  policyengine:
-    replicas: 1
-
-  producer:
-    ceph_le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    cos_le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    cos_scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    file_scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    protect_scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    scale_le:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-    scale_scan:
-      batch_size: 50000
-      outstream_config_list: none
-      instream_config_list: none
-      debug: none
-      replicas: 1
-
-  scaleafmdatamover:
-    replicas: 1
-
-  scaleilmdatamover:
-    replicas: 1
-
-  sdmonitor:
-    replicas: 1
-
-  tikaserver:
-    replicas: 1
-
-  ui:
-    frontend:
-      replicas: 1
-    backend:
-      replicas: 1
-
-  wkcconnector:
-    replicas: 1
-
+  rwx_storage_class: ${ISD_SC_RWX}
+  rwo_storage_class: ${ISD_SC_RWO}
```

Comment:
This is not true, since the rwo_storage_class is something optional and this can be interpreted as we require this parameter. Anyway, we are providing this file in the documentation as discussed by the team, and having this also in the public repo could be redundant, so is a better idea just remove it from here.

Reply:
I agree, it makes no sense to mimic the documentation here, I'll remove the file